recipes/r-bmisc/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = '1.4.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bmisc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BMisc_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BMisc/BMisc_{{ version }}.tar.gz
+  sha256: 304095a6b0e8e1b51dd411215d613915c5cee441aa131f666d3fb1078bea38c8
+
+build:
+  merge_build_host: True  # [win]
+  missing_dso_whitelist:
+    - '*/R.dll'            # [win]
+    - '*/Rblas.dll'        # [win]
+    - '*/Rlapack.dll'      # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```

I would suggest just to skip windows for now. We are switching to UCRT-based builds and moving to R 4.3/4.4, whereas this is testing the old build tools on R 4.1.

```suggestion
  skip: true  # [win]

```